### DISCUSSION TITLE
[Feature] Multiple APIs in config

### DISCUSSION THREAD

**Discussion by tehniemer** [damongolding/immich-kiosk#196] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'd love it if I could enter multiple API keys so immich-kiosk could access photos across multiple accounts.

Scenario: my wife and I both take photos of and with our children, it would be wonderful if I could add both of our account API keys and the people IDs of our children from each account to create a shared, auto updating, slideshow.

**Comment by damongolding** [comment]
This request feels similar to [#95](https://github.com/damongolding/immich-kiosk/discussions/95). The notion of multiple api keys has been discussed before, but the implementation would require huge changes to the Kiosk code base.

**Reply by tehniemer** [reply]
It is similar, but not the same. That issue seems to be geared towards shared albums with a single API.

**Comment by damongolding** [comment]
Once the Immich team rolls out smart albums, I believe this functionality will become unnecessary. With smart albums, you could create an album, share it with your wife, and both of you could add people to it. The album would automatically update with new images, making it easier to manage. You could then simply link that album to the Kiosk.

**Reply by tehniemer** [reply]
That was my thought as well, but I'm not sure it's on the roadmap yet. I was hoping this solution would be fairly easy to implement here for the interim, but it sounds like it isn't that simple, so I can be patient and wait for the feature in immich.
